yourgov/components/StaffMembersTable/FilterSearchInput.tsx

```diff
@@ -0,0 +1,25 @@
+import { SearchInput } from '@ag.ds-next/react/src/search-input';
+import { useSortAndFilterContext } from './lib/contexts';
+import { tableId } from './DataTable';
+
+export const FilterSearchInput = ({ block }: { block?: boolean }) => {
+	const { filters, setFilter } = useSortAndFilterContext();
+
+	return (
+		<div role="search">
```

Comment:
Need to an `aria-label` again (or `aria-labelledby`)
```suggestion
		<div aria-label="Staff members" role="search">
```